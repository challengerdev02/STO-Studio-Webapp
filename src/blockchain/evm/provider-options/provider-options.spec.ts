import { cleanup } from '@testing-library/react';
import { providerOptions } from './index';

describe('ProviderOptions object', () => {
  afterEach(cleanup);

  const setup = () => {
    return {
      ...providerOptions,
    };
  };

  it('should test providerOptions', function () {
    const res = setup();

    expect(res).toBeTruthy();
    expect(res.bsc).toBeTruthy();
    expect(res.authereum).toBeTruthy();
    expect(res.fortmatic).toBeTruthy();
    expect(res.frame).toBeTruthy();
    expect(res.portis).toBeTruthy();
  });

  it('should match with snapshot', () => {
    // @ts-ignore
    const { container } = setup();
    expect(container).toMatchSnapshot();
  });
});
