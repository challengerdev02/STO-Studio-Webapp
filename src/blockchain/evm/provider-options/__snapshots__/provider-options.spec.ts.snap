// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProviderOptions object should match with snapshot 1`] = `undefined`;
